src/api/java/com/minecolonies/api/util/ItemStackUtils.java

```diff
@@ -312,6 +314,20 @@ else if (ToolType.HELMET.equals(toolType)
                 return getArmorLevel(ArmorItem.getArmorMaterial());
             }
         }
+        else if (stack.getItem() instanceof FishingRodItem)
+        {
+            if(stack.getItem().isDamageable())
+            {
+                if(stack.getMaxDamage() < DURABILITY_MAX_WOOD_OR_GOLD)
```

Comment:
Isn't there a registry where we could get that dynamically? Like maybe getting another tool and comparing the durability to make this more dynamic?

Reply:
But because Fishing Rods are used much slower than other tools, Aquaculture gives them lower max durability: an Aquaculture Diamond Fishing Rod's max durability is 450 to the 1561 of vanilla's Diamond Pick/Axe/Shovel/Sword, and even a Neptunum Fishing Rod only goes to 1000.  The disparity is less extreme for iron, but at 125 the Aquaculture Iron Fishing Rod has lower durability than a vanilla Stone Pickaxe's 131.

Thermal Foundation's custom fishing rods more closely match their vanilla equivalents, but even they are a little off (TF Iron Fishing Rods at 25_5_, for example). I picked 400 so that steel and invar fishing rods (https://teamcofh.com/docs/1.12/thermal-foundation/fishing-rods/) would require a higher tier hut, for example, but that is admittedly a magic number.

Comparing to multiples of the vanilla fishing rod would better handle situations where someone just buffs the vanilla one, but net.minecraft.Items.FISHING_ROD.getMaxDurability() is deprecated.  net.minecraft.item.ItemTier would normally be great, but vanilla fishing rods don't use it to determine their uses or match its pattern.

I suppose using ItemTier's getMaxUses, adjusting it, and leaving a comment about the magic numbers to squeeze an offset in is probably the best option.